services/business/workloads.go

```diff
@@ -31,3 +31,16 @@ func (in *WorkloadService) GetWorkloadList(namespace string) (models.WorkloadLis
 	}
 	return *workloadList, nil
 }
+
+// GetWorkload is the API handler to fetch details of an specific workload
+func (in *WorkloadService) GetWorkload(namespace string, workloadName string) (*models.Workload, error) {
+	deployment, err := in.k8s.GetDeploymentDetails(namespace, workloadName)
+	if err != nil {
+		return &models.Workload{}, err
+	}
+
+	model := &models.Workload{}
+	model.Parse(deployment.Deployment)
```

Comment:
Quite minor, one could think that model.Parse() and model.SetDetails() could be done at same time, as basically it's same model.
Not a blocker or action needed, it just comes to my attention, but I guess when I see all code I can understand it better.